modules/Rock.cmake

```diff
@@ -457,26 +457,34 @@ macro(rock_target_definition TARGET_NAME)
                 get_filename_component(__file_wext ${__moced_file} NAME_WE)
                 get_filename_component(__file_dir ${__moced_file} PATH)
                 if (NOT "${__file_dir}" STREQUAL "")
-		    set(__file_wext "${__file_dir}/${__file_wext}")
+            set(__file_wext "${__file_dir}/${__file_wext}")
                 endif()
-		unset(__moced_file)
-		foreach(__header_ext .h .hh .hxx .hpp)
-		    if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${__file_wext}${__header_ext}")
-			set(__moced_file "${__file_wext}${__header_ext}")
-		    endif()
-		endforeach()
+        unset(__moced_file)
+        foreach(__header_ext .h .hh .hxx .hpp)
+            if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${__file_wext}${__header_ext}")
+            set(__moced_file "${__file_wext}${__header_ext}")
+            endif()
+        endforeach()
```

Comment:
You converted tabs to spaces, but did so while your editor was configured to use 4 spaces per tab. Indentation is still broken. Please double-check on the GH pull request interface (which uses 8-spaces tabs)

Reply:
I only highlighted this, but there are other places with the same problem.

Reply:
@doudou Sorry for that, have been to inattentive with it. Now I cannot see any wrongly indented lines in the Github diff for the PR.